ddl/ddl.go

```diff
@@ -204,6 +219,46 @@ type ddl struct {
 	sessPool          *sessionPool
 	delRangeMgr       delRangeManager
 	enableTiFlashPoll *atomicutil.Bool
+	// used in the concurrency ddl.
+	reorgWorkerPool      *workerPool
+	generalDDLWorkerPool *workerPool
+	// get notification if any DDL coming.
+	ddlJobCh chan struct{}
+}
+
+// waitSchemaSyncedController is to control whether to waitSchemaSynced or not.
+type waitSchemaSyncedController struct {
+	mu  sync.RWMutex
+	job map[int64]struct{}
+
+	// true if this node is elected to the DDL owner, we should wait 2 * lease before it runs the first DDL job.
+	once *atomicutil.Bool
+}
+
+func newWaitSchemaSyncedController() *waitSchemaSyncedController {
+	return &waitSchemaSyncedController{
+		job:  make(map[int64]struct{}, 16),
```

Comment:
Add a constant to this?  Also, I see that `runningJobIDs` is the same value, let's add a constant together?